### ISSUE TITLE
Have automatic update of tenants

### ISSUE THREAD

**Issue by maxandersen** [openshiftio/openshift.io#242]
automatic updates of tenants are not working at the moment making it non managable when we start rolling out new versions and updates. 

We'll need to either automate this or at least warn the user to do a manual update.

**Comment by joshuawilson** [comment]
Who owns this? @aslakknutsen or someone else?

**Comment by maxandersen** [comment]
yes, its for @aslakknutsen but he haven't accepted invite so he can be assigned :)

**Comment by jstrachan** [comment]
before we have automatic updates; we need to get updates working at all ;) they seem to be pretty flaky -
 I've yet to have one work OOTB for me without manual hacks to work around issues: https://github.com/fabric8io/fabric8-init-tenant/issues/74

**Comment by sbose78** [comment]
I really find it necessary to expose the 'latest version' as an API endpoint. of our tenant apps ( the prod version of https://github.com/fabric8io/fabric8-init-tenant/blob/master/TEAM_VERSION ) to the user.

Proposal : https://github.com/almighty/almighty-core/pull/1410

That way there's a way to show the user if even the update fails "which version you are on.." and "which version is the latest..".

**Comment by jstrachan** [comment]
we need a way to phase the rollout of new tenant versions as part of the CD pipeline - see https://github.com/fabric8io/fabric8-init-tenant/issues/68

**Comment by joshuawilson** [comment]
@burrsutter do you want this on the Workshop list?

**Comment by aslakknutsen** [comment]
Without any EE tests running this is a very quick way to roll our bugs to everyone.

**Comment by joshuawilson** [comment]
yeah, more bugs

OK, priority is to get EE tests first. To be clear, they are on the update tenant.

**Comment by aslakknutsen** [comment]
>  To be clear, they are on the update tenant.
That wasn't clear at all. What?

**Comment by joshuawilson** [comment]
You said we need EE test on an issue about updating the tenant. Currently we are running the EE tests but not as a part of the build and we are adding them to the UI part of the build. Neither of those will prevent a bad tenant update from being deployed. Therefore we need to get the EE tests to be added to the update tenant, which was discussed on https://github.com/openshiftio/openshift.io/issues/435.

**Comment by joshuawilson** [comment]
@aslakknutsen is this done or still in-progress?
